spec/features/collection_drafts/forms/acquisition_information_form_spec.rb

```diff
@@ -4,6 +4,19 @@
     draft = create(:collection_draft, user: User.where(urs_uid: 'testuser').first)
     visit collection_draft_path(draft)
   end
+  
+  context 'when checking the accordion headers for required icons' do
+    before do
+      within '.metadata' do
```

Comment:
Since this is shared across both contexts, this can be moved into the describe's before block.

Reply:
Okay will do - there are a lot of files like this, so I can make sure they're all adjusted